### PR TITLE
[Cu-29vke38] Entity tokens per underlying asset type

### PR DESCRIPTION
**Pull request by amarinkovic**
### Summary

This PR adds support for creating multiple types of tokens per entity. Different entity tokens will be produced for each of the underlying assets.

ClickUp: [#29vke38](https://app.clickup.com/t/29vke38)

### CODE REVIEW

**Comment by melks** [comment]
Task linked: [CU-29vke38 Add support for entity tokens for each underlying asset](https://app.clickup.com/t/29vke38)

File: contracts/EntityFundingFacet.sol [hunk 1]
```diff
@@ -8,6 +8,7 @@ import "./base/IDiamondFacet.sol";
 import "./base/IERC20.sol";
 import "./base/IMarket.sol";
 import "./EntityFacetBase.sol";
+import "hardhat/console.sol";
```
Comment (kevin-fruitful) [inline contracts/EntityFundingFacet.sol:11]: should remove test tooling prior to live deployments

Comment (amarinkovic) [reply contracts/EntityFundingFacet.sol:11]: Yes, of course, I though I did remove it 🤦🏻‍♂️

Comment (amarinkovic) [reply contracts/EntityFundingFacet.sol]: done

**Pull request merged**